docs/source/api.rst

```diff
@@ -509,3 +509,10 @@ PipelineStatusUpdateError
 
 .. autoclass:: neo4j_graphrag.experimental.pipeline.exceptions.PipelineStatusUpdateError
    :show-inheritance:
+
+
+InvalidJSONError
```

Comment:
I think the exceptions are also listed above, though I don't know why we have this list :)

Reply:
Do you mean the hyperlinks listed above?

Reply:
Yes, thank you!